spec/models/league_spec.rb

```diff
@@ -58,12 +61,20 @@
       end
     end
     
-      it "does only let half as many matches as teams play per gameday" do
-        5.times do |gameday|
-          gameday+=1 #gamedays are from 1 to 5 not 0 to 4
-          gameday_matches = matches.select{|match| match.gameday==gameday}
-          expect(gameday_matches.length).to be 2
-        end
+    it "does only let half as many matches as teams play per gameday" do
+      5.times do |gameday|
+        gameday+=1 #gamedays are from 1 to 5 not 0 to 4
+        gameday_matches = matches.select{|match| match.gameday==gameday}
+        expect(gameday_matches.length).to be 2
       end
+    end
+
+    it "double round robin has double the gamedays as normal round robin" do
+      new_league = FactoryBot.create(:league_with_teams)
+      new_league.calculate_double_round_robin
+      new_matches = new_league.matches
+
+      expect(new_matches.length).to be new_league.teams.length * (new_league.teams.length - 1)
```

Comment:
you aren't actually checking for gamedays but just for how many matches are created. Maybe change the test to be a bit more precise (or add a few more)